pkg/scheduler/metrics/metrics.go

```diff
@@ -192,6 +192,16 @@ var (
 			Help:      "Total preemption attempts in the cluster till now",
 		})
 
+	pendingPods = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Subsystem: SchedulerSubsystem,
+			Name:      "pending_pods_total",
```

Comment:
The total suffix is meant for counter according to the metrics instrumentation guidelines. This should just be “pending_pods”

Reply:
Sure. I will raise a follow up PR shortly. We do need sig-instrumentation 's advice on best practices, conventions on this kind of PR :-)

Thanks @brancz .